### PR TITLE
Rename Embedded Wallets -> In-App Wallets

### PR DESCRIPTION
**Pull request by ianthirdweb**
## PR-Codex overview
This PR updates the terminology from "Embedded Wallet" to "In-App Wallet" across various files and components.

### Detailed summary
- Renamed "Embedded Wallet" to "In-App Wallet" for consistency
- Updated wallet labels and URLs
- Replaced references to "Embedded Wallet" with "In-App Wallet"

> The following files were skipped due to too many changes: `src/app/connect/account-abstraction/guides/react/page.mdx`, `src/app/unity/wallets/actions/getemail/page.mdx`, `src/app/typescript/v5/why-thirdweb/page.mdx`, `src/app/account/billing/account-info/page.mdx`, `src/app/account/api-keys/edit-services/page.mdx`, `src/app/connect/embedded-wallet/how-to/get-embedded-wallet-details-on-server/page.mdx`, `src/app/solutions/gaming/overview/page.mdx`, `src/app/typescript/v5/adapters/page.mdx`, `src/app/wallet-sdk/v2/page.mdx`, `src/app/connect/embedded-wallet/how-to/export-private-key/page.mdx`, `src/app/connect/account-abstraction/faq/page.mdx`, `src/app/payments/nft-checkout/faq/page.mdx`, `src/app/account/api-keys/page.mdx`, `src/app/connect/page.mdx`, `src/app/unity/wallets/actions/connect/page.mdx`, `src/app/connect/account-abstraction/get-started/page.mdx`, `src/app/connect/embedded-wallet/how-to/interact-blockchain/page.mdx`, `src/app/connect/embedded-wallet/how-to/connect-users/page.mdx`, `src/app/connect/embedded-wallet/how-to/interact-with-wallets/page.mdx`, `src/app/connect/in-app-wallet/custom-auth/overview/page.mdx`, `src/app/unity/wallets/providers/embedded-wallet/page.mdx`, `src/app/connect/embedded-wallet/custom-auth/custom-jwt-auth-server/page.mdx`, `src/app/account/billing/credits/page.mdx`, `src/app/connect/embedded-wallet/how-to/enable-gasless/page.mdx`, `src/app/connect/sidebar.tsx`, `src/app/connect/in-app-wallet/overview/page.mdx`, `src/app/connect/account-abstraction/how-it-works/page.mdx`, `src/app/wallet-sdk/v2/build/connect-wallet-integration/page.mdx`, `src/app/connect/embedded-wallet/how-it-works/page.mdx`, `src/app/connect/embedded-wallet/get-started/page.mdx`, `src/app/connect/embedded-wallet/custom-auth/custom-auth-server/page.mdx`, `src/app/connect/embedded-wallet/how-to/build-your-own-ui/page.mdx`, `src/app/connect/embedded-wallet/custom-auth/firebase-auth/page.mdx`, `src/app/react-native/v0/wallets/embedded-wallet/page.mdx`, `src/app/connect/embedded-wallet/faqs/page.mdx`, `src/app/connect/embedded-wallet/custom-auth/configuration/page.mdx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by 0xFirekeeper — DISMISSED** [review]

File: src/app/connect/in-app-wallet/custom-auth/configuration/page.mdx [hunk 1]
```diff
@@ -48,24 +48,28 @@ You will be asked to enter the following values
...
+const { connect } = useConnect();
 
 const handlePostLogin = async (jwt: string) => {
-	await embeddedWallet.connect({
-		strategy: "jwt",
-		jwt,
-		encryptionKey: "your-encryption-key",
+	await connect(() => {
+		const wallet = inAppWallet();
+		wallet.connect({
```
Comment (MananTank) [inline src/app/connect/in-app-wallet/custom-auth/configuration/page.mdx:66]: add await here

File: src/app/connect/in-app-wallet/custom-auth/configuration/page.mdx [hunk 2]
```diff
@@ -136,29 +135,33 @@ You can also pass a list of headers. These headers will be sent with every reque
...
 
 const handlePostLogin = async (jwt: string) => {
-	await embeddedWallet.connect({
-		strategy: "auth_endpoint",
-		// This is the payload that is sent to the auth endpoint
-		payload,
-		encryptionKey: "your-encryption-key",
+	await connect(() => {
+		const wallet = inAppWallet();
+		wallet.connect({
```
Comment (MananTank) [inline src/app/connect/in-app-wallet/custom-auth/configuration/page.mdx:157]: add await

File: src/app/connect/in-app-wallet/custom-auth/custom-auth-server/page.mdx [hunk 3]
```diff
@@ -94,19 +94,22 @@ You can now auth into the wallet and use it to sign transactions like so (see [u
...
-		encryptionKey: "your-encryption-key",
+import { inAppWallet } from "thirdweb/wallets";
+import { useConnect } from "thirdweb/react";
+
+const { connect } = useConnect();
+
+const handlePostLogin = async (jwt: string) => {
+	await connect(() => {
+		const wallet = inAppWallet();
+		wallet.connect({
```
Comment (MananTank) [inline src/app/connect/in-app-wallet/custom-auth/custom-auth-server/page.mdx:106]: add await

File: src/app/connect/in-app-wallet/custom-auth/custom-jwt-auth-server/page.mdx [hunk 4]
```diff
@@ -6,17 +6,17 @@ import EWJWKCreatorTool from "../images/jwk-creator-tool.png";
 
 export const metadata = createMetadata({
 	image: {
-		title: "Create a Custom JWT Auth Server for Embedded Wallet",
+		title: "Create a Custom JWT Auth Server for In-App Wallet",
 		icon: "wallets",
 	},
-	title: "Create a Custom JWT Auth Server | EmbeddedWallet",
+	title: "Create a Custom JWT Auth Server | InAppWallet",
```
Comment (MananTank) [inline src/app/connect/in-app-wallet/custom-auth/custom-jwt-auth-server/page.mdx:12]: Use "In-App wallet" here

**Review by MananTank — CHANGES_REQUESTED** [review]
This overrides a lot of changes in main - looks like main was not properly merged in. Let's start a new PR

File: src/app/typescript/v5/react/wallets/page.mdx [hunk 5]
```diff
@@ -0,0 +1,92 @@
+import { Breadcrumb, createMetadata } from "@doc";
```
Comment (MananTank) [inline src/app/typescript/v5/react/wallets/page.mdx:1]: For example: This page has been deleted and replaced with a different page - and this PR adds it back

Comment (joaquim-verges) [reply src/app/typescript/v5/react/wallets/page.mdx]: deleted

**Review by joaquim-verges — APPROVED** [review]

**Pull request merged**